src/chrome/chromeDebugAdapter.ts

```diff
@@ -859,6 +935,10 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         return this.setBreakpoints(pendingBP.args, pendingBP.requestSeq, pendingBP.ids).then(response => {
             response.breakpoints.forEach((bp, i) => {
                 bp.id = pendingBP.ids[i];
+                // If any of the unbound breakpoints in this file is on (1,1), we set userBreakpointOnLine1Col1 to true
+                if (bp.line === 1 && bp.column === 1) {
```

Comment:
@roblourens Given that Chrome tends to ignore column numbers, is this the best check to do here?